src/lib/types/events.ts

```diff
@@ -248,6 +250,7 @@ export const IEventTypes = [
     CHANGE_REQUEST_CANCELLED,
     CHANGE_REQUEST_SENT_TO_REVIEW,
     CHANGE_REQUEST_APPLIED,
+    SCHEDULED_CHANGE_REQUEST_APPLIED,
     CHANGE_REQUEST_SCHEDULED,
```

Comment:
This helps the enterprise notification service create a more meaningful message.  
Considered doing it by keeping the same event type CHANGE_REQUEST_APPLIED and adding `scheduled` param in ChangeRequestApplied event in enterprise.
Decided that having a separate event made more sense
(eg one difference is that CHANGE_REQUEST_APPLIED event does not create a notification for the 'applier'   whereas SCHEDULED_CHANGE_REQUEST_APPLIED will)